client/src/Layout/Footer.tsx

```diff
@@ -0,0 +1,178 @@
+import { Box, Container, Grid2, Typography ,Divider} from "@mui/material"
+import React from "react"
+import Input from "../component/Input"
+import SendIcon from "@mui/icons-material/Send"
+import Qrimagefooter from "../image/Frame 719.png"
+import FacebookIcon from "@mui/icons-material/Facebook"
+import InstagramIcon from "@mui/icons-material/Instagram"
+import TwitterIcon from "@mui/icons-material/Twitter"
+import LinkedInIcon from "@mui/icons-material/LinkedIn"
+import { styled } from '@mui/system'
+
+const FooterBox = styled(Box)`
+  background-color: black;
+  padding: 30px 0px 30px 0px;
+
+`
+const FooterInput = styled(Box)`
+  border: 1px solid white;
+  width: 210px;
+  height: 46px;
+  margin-top: 16px;
+`
```

Comment:
Please replace to `const FooterInput = styled()(({ theme }) => ({

}))`

Reply:
all the styled component is change into theme arrow function